src/main/java/io/goobox/sync/storj/StorjUtil.java

```diff
@@ -71,4 +73,16 @@ public static boolean isTemporaryError(int code) {
         }
     }
 
+    public static boolean isExcluded(Path path) throws IOException {
```

Comment:
It would be better if you catch the IOException here instead of re-throwing it. If you catch IOException, perhaps return true, i.e. to exclude the file from sync. If the file size cannot be determined then most probably the sync will fail too, so better exclude that file.

Reply:
yea, sure